### PR TITLE
Made changes to highlight new DT UI features

### PR DESCRIPTION
**Pull request by rhetoric101**
DT now has some new filters, grouping, and charts. I added some documentation for each of these in the DT UI doc.

### CODE REVIEW

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx [hunk 1]
```diff
@@ -133,30 +130,47 @@ Here are two query bar examples:
...
+The default view of distributed tracing shows traces grouped by the same root entry span. In other words, traces are grouped by the span where New Relic began recording the request. You can slide the toggle **Group similar traces** to turn this on and off.
+
+With trace groups you get a high-level view of traces so you can understand request behavior for groups of similar traces. This is extremely helpful to understand dips or spikes in trace count, duration, and errors. 
+
+When you click on one of the trace groups, you get all the standard details in context of the specific trace group you selected.
+
 ### Filter using the scatter plot [#scatter-plot]
 
-The trace scatter plot on the opening page of distributed tracing is a quick way to search for outlying traces. You can move the cursor across the chart to view trace details and you can click individual points to get details.
+The trace scatter plot is a quick way to search for outlying traces. This is is available on the opening page of distributed tracing if you turn off **Group similar traces** toggle at the top of the page.
```
Comment (urbiz-nr) [inline src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:143]: turn off _the_ **Group similar traces**

**Review by urbiz-nr — DISMISSED** [review]
Nice updates!

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:143]: @urbiz-nr, good suggestion!

**Comment by rhetoric101** [comment]
Reviewer: This PR was originally part of Task DOC-6850 (Add documentation for new trace filters, trace groups, and golden signals charts), but there was a delay in finalizing the content before the end of the sprint.

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx [hunk 2]
```diff
@@ -133,30 +130,43 @@ Here are two query bar examples:
   </Collapser>
 </CollapserGroup>
 
+### Trace groups [#trace-groups]
+
+The default view of distributed tracing shows traces grouped by the same root entry span. In other words, traces are grouped by the span where New Relic began recording the request. You can slide the toggle **Group similar traces** to turn this on and off.
+
+With trace groups you get a high-level view of traces so you can understand request behavior for groups of similar traces. This is helps you understand dips or spikes in trace count, duration, and errors.
```
Comment (urbiz-nr) [inline src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:137]: Remove "is" after "This" in the second sentence!

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx [hunk 3]
```diff
@@ -133,30 +130,43 @@ Here are two query bar examples:
...
+The default view of distributed tracing shows traces grouped by the same root entry span. In other words, traces are grouped by the span where New Relic began recording the request. You can slide the toggle **Group similar traces** to turn this on and off.
+
+With trace groups you get a high-level view of traces so you can understand request behavior for groups of similar traces. This is helps you understand dips or spikes in trace count, duration, and errors. 
+
+When you click on one of the trace groups, you get all the standard details in context of the specific trace group you selected.
+
 ### Filter using the scatter plot [#scatter-plot]
 
-The trace scatter plot on the opening page of distributed tracing is a quick way to search for outlying traces. You can move the cursor across the chart to view trace details and you can click individual points to get details.
+The trace scatter plot is a quick way to search for outlying traces. This is is available on the opening page of distributed tracing if you turn off the **Group similar traces** toggle at the top of the page.
```
Comment (urbiz-nr) [inline src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:143]: There's to "is" in the second sentence.

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx [hunk 4]
```diff
@@ -133,30 +130,43 @@ Here are two query bar examples:
...
+1. In the **View by** dropdown, select the duration type: 
+    * **Back-end duration**
+    * **Root span duration**
+    * **Trace duration**
+2. In **Group traces by**, select one of these options:
+    * **Errors**: Group by whether or not traces contain errors.
+    * **Root service**: Group by the name of the first service in traces. In a trace where Service A calls Service B and Service B calls Service C, the root service would be Service A.
+    * **Root entry span**: Group by the root transaction, which is the root service's endpoint. In a trace where Service A calls Service B and Service B calls Service C, the root entry span is Service A's endpoint. For example: "Service A - GET /user/%".
+    * **Service entry span**: Group by the span name of the service currently being viewed in APM. For example, for a trace where Service A calls Service B and Service B calls Service C, if you are viewing Service B in APM and select this grouping, the traces will be represented by their Service B span names. If a service has multiple spans in a trace, this grouping option will use that service's first entry point.
```
Comment (urbiz-nr) [inline src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:159]: you're viewing

**Review by urbiz-nr — CHANGES_REQUESTED** [review]
Great updates. Please check the typos!

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:137]: @urbiz-nr, good catch!

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx (hunk 4 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:159]: @urbiz-nr, I changed this. Thanks!

File: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx (hunk 3 above)
Comment (rhetoric101) [reply src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx:143]: @urbiz-nr, good catch! Thank you!

**Pull request merged**